system/client/src/pages/UploadsPage/UploadsPage.tsx

```diff
@@ -18,7 +18,7 @@ function UploadsPage() {
 
   const [data, setData] = useState<DataType[] | [] >([]);
 
-  // const data = [
+  // const data = [ 

```

Comment:
This file is mostly commented code. This should be cleaned up before submitting a technical review